noir-projects/aztec-nr/aztec/src/context/public_context.nr

```diff
@@ -356,6 +374,9 @@ unconstrained fn l1_to_l2_msg_exists_opcode(msg_hash: Field, msg_leaf_index: Fie
 #[oracle(avmOpcodeSendL2ToL1Msg)]
 unconstrained fn send_l2_to_l1_msg_opcode(recipient: EthAddress, content: Field) {}
 
+#[oracle(avmOpcodeCalldataCopy)]
+unconstrained fn calldata_copy_opcode<let N: u32>(cdoffset: u32, copy_size: u32) -> [Field; N] {}
+
```

Comment:
Why do we need this oracle now, but didn't need it before? There is no longer just a simple 1-1 mapping between the calldatacopy brillig opcode and the avm opcode?